README.md

```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
-`/examples/apps/cli`. See the
-[CLI example README](examples/apps/cli/README.md)
-for more details.
+Other OpenThread-specific features and enhancements include:
 
+* Network authentication with Commissioning
+* IPv6 configuration and raw data interface
+* UDP sockets
+* CoAP client and server
+* DHCPv6 client and server
+* DNSv6 client
+* Command Line Interface (CLI)
+* Network Co-Processor (NCP) support - off-loading Thread functions to a SoC
+* Spinel - general purpose NCP protocol
+* `wpantund` - user-space NCP network interface driver/daemon
+* Sniffer support via NCP Spinel nodes
 
-## What's included ##
+### What's coming?
 
-In the repo you'll find the following directories and files:
+The development of OpenThread is ongoing to regularly provide additional features not available in the standard. Currently, we are working on:
 
-File/Folder   | Provides
---------------|----------------------------------------------------------------
-`doc`         | Code and protocol docs
-`etc`         | Configuration files for other build systems (e.g. Visual Studio)
-`examples`    | Sample applications and platforms demonstrating OpenThread
-`include`     | Public API header files
-`src`         | Core implementation of the Thread standard and related add-ons
-`tests`       | Unit and Thread conformance tests
-`third_party` | Third-party code used by OpenThread
-`tools`       | Helpful utilities related to the OpenThread project
+* Host-side Border Agent support
+
+# Who supports OpenThread?
+
+Led by Nest, the following companies are contributing to the ongoing development of OpenThread:
+
+[![ARM][arm-logo]][arm-link][![Atmel][atmel-logo]][atmel-link][![Dialog][dialog-logo]][dialog-link][![Microsoft][ms-logo]][ms-link][![Nest][nest-logo]][nest-link][![Nordic][nordic-logo]][nordic-link][![Qualcomm][qc-logo]][qc-link][![Texas Instruments][ti-logo]][ti-link]
+
+OpenThread has been ported to several devices and platforms. Build examples are included in the OpenThread library.
+
+[arm-logo]: doc/images/ot-contrib-arm.png
+[arm-link]: https://www.arm.com/
+[atmel-logo]: doc/images/ot-contrib-atmel.png
+[atmel-link]: http://www.atmel.com/
+[dialog-logo]: doc/images/ot-contrib-dialog.png
+[dialog-link]: http://www.dialog-semiconductor.com/
+[ms-logo]: doc/images/ot-contrib-ms.png
+[ms-link]: https://www.microsoft.com/en-us/
+[nest-logo]: doc/images/ot-contrib-nest.png
+[nest-link]: https://nest.com/
+[nordic-logo]: doc/images/ot-contrib-nordic.png
+[nordic-link]: http://www.nordicsemi.com/
+[qc-logo]: doc/images/ot-contrib-qc.png
+[qc-link]: https://www.qualcomm.com/
+[ti-logo]: doc/images/ot-contrib-ti.png
+[ti-link]: https://www.ti.com/
+
+### IEEE 802.15.4 Platform Support
 
+* [POSIX Emulation](https://github.com/openthread/openthread/wiki/Platforms#posix-emulation)
+* [Dialog Semiconductor DA15000 OpenThread Sandbox Development Kit](https://github.com/openthread/openthread/wiki/Platforms#dialog-da15000)
```

Comment:
@jwhui Where should I mention POSIX Emulation then... as its own independent thing or under Desktop Support?